src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-message-thread-details-expand-all.cypress.spec.js

```diff
@@ -41,6 +41,8 @@ describe('Secure Messaging Message Details', () => {
     );
     cy.get('@allMessageDetails.all').should('have.length', 0);
     cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT);
```

Comment:
The temporary fix for this [card](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2952) to improve the experience for speech recognition users was flagged for and axe check violation. This change disables that rule.